Sail/src/Sail/utils/SailImGui/OptionsWindow.cpp

```diff
@@ -164,3 +191,202 @@ bool OptionsWindow::renderGameOptions() {
 
 	return settingsChanged;
 }
+
+void OptionsWindow::drawCrosshair() {
+	// Fetch current settings from settingsStorage
+	auto& stat = m_settings->applicationSettingsStatic;
+	auto& dynamic = m_settings->applicationSettingsDynamic;
+	float screenWidth = 100.0f;
+	float screenHeight = 100.0f;
+
+	screenWidth = ImGui::GetCurrentWindow()->Pos.x + 5;
+	screenHeight = ImGui::GetCurrentWindow()->Pos.y + 5;
+
+	float thickness = dynamic["Crosshair"]["Thickness"].value;
+	float centerPadding = dynamic["Crosshair"]["CenterPadding"].value;
+	float size = dynamic["Crosshair"]["Size"].value;
+	float outerAlteredPadding = dynamic["Crosshair"]["OuterAlteredPadding"].value;
```

Comment:
Could temporarily store
    auto& crosshairSetting = dynamic["Crosshair"];
to lower map lookups

Reply:
Fixed